### DISCUSSION TITLE
DB25 pinout

### DISCUSSION THREAD

**Discussion by shift838** [ZuluSCSI/ZuluSCSI-firmware#417] [Q&A] [answered]
Looking for the DB25 pinout for the zuluscsi so I can make my own cable for a older system that uses the DB25 but not the standard wiring.

I have not been able to find it.

Can you point me to a link or provide the pinout to the ZuluSCSI.  I am assuming all the ZuluSCSI interfaces use the same DB25 pinout?  I have a few of different ZuluSCSI revisions that use different microcontrollers but all have a DB25 connector.

**Comment by aperezbios** [comment, marked as answer]
@shift838:
```
External SCSI Connector (DB25 Macintosh-Type Female Pseudo-SCSI)
   Pin 1: *REQ
   Pin 2: *MSG
   Pin 3: *I/O
   Pin 4: *RST
   Pin 5: *ACK
   Pin 6: *BSY
   Pin 7: Ground
   Pin 8: *Data 0
   Pin 9: Ground
  Pin 10: *Data 3
  Pin 11: *Data 5
  Pin 12: *Data 6
  Pin 13: *Data 7
  Pin 14: Ground
  Pin 15: *C/D
  Pin 16: Ground
  Pin 17: *ATN
  Pin 18: Ground
  Pin 19: *SEL
  Pin 20: *Parity
  Pin 21: *Data 1
  Pin 22: *Data 2
  Pin 23: *Data 4
  Pin 24: Ground
  Pin 25: Terminator Power
```

**Reply by shift838** [reply]
thank you..
